src/pages/calculator.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+// eslint-disable-next-line import/extensions
+import calculate from '../Logic/calculate.js';
```

Comment:
- [ ] There's an error while running your application due to the wrong file being imported. It should be `'../logic/calculate'`. Kindly fix this 🙏 

![image](https://user-images.githubusercontent.com/74814780/206534137-989d4c29-2819-4791-acce-ff1f8ff42154.png)


- [ ] [OPTIONAL] When importing a `js` file there's no need to add a `.js` extension as it imports all files as `js` by default. Kindly fix this 🙏

Reply:
Thank you